src/app/pages/AccountPage.jsx

```diff
@@ -36,16 +37,15 @@ const AccountPage = (props) => {
 
   const [isLoading, setIsLoading] = useState(true);
   const [itemToCancel, setItemToCancel] = useState(null);
+  const [displayModal, setDisplayModal] = useState(false);
```

Comment:
I actually kept confusing `AccountPage` and `AccountPageModal` and when I left the previous comment forget which I was looking at. There are actually two modals for the account page - this one and the hold cancellation. Can we call the component `TimedLogoutModal` or `LogoutModal` and call this `display[component name]`?

Reply:
I've renamed everything to use `TimedLogoutModal` now. The diff may look bigger than it actually is -- only the name has been changed.